modules/ocf_mirrors/manifests/centos_altarch.pp

```diff
@@ -0,0 +1,26 @@
+class ocf_mirrors::centos_altarch {
+  file { '/opt/mirrors/project/centos-altarch':
+    ensure  => directory,
+    source  => 'puppet:///modules/ocf_mirrors/project/centos-altarch/',
+    owner   => mirrors,
+    group   => mirrors,
+    mode    => '0755',
+    recurse => true,
+  }
+
+  ocf_mirrors::monitoring {
+    'centos-altarch':
+      type          => 'unix_timestamp',
+      upstream_host => 'mirror.centos.org',
+      upstream_path => '/altarch',
+      ts_path       => 'TIME';
+  }
+
+  cron {
+    'centos-altarch':
+      command => '/opt/mirrors/project/centos-altarch/sync-altarch > /dev/null',
+      user    => 'mirrors',
+      hour    => '*/3',
+      minute  => '44';
+  }
+}
```

Comment:
I think ideally we'd make a new class to simplify adding new mirrors, since there's a lot of duplicated content here (e.g. file owners, paths, etc.) that we could extract and put into a common class that handles adding a whole mirror with monitoring. Anyway, that's probably content for a future PR, not this one.